index.bs

```diff
@@ -8,14 +8,20 @@ Editor: Michael Blain, Google, mpb@google.com
 Abstract: This specification describes  an API that can be used to understand the visibility and position of DOM elements relative to a viewport. The position is delivered asynchronously and is useful for understanding the visibility of elements and implementing pre-loading and deferred loading of DOM content.
 Group: Web Performance Working Group
 Repository: slightlyoff/IntersectionObserver
-Mailing List: public-web-perf@w3c.org
-Mailing List Archives: http://lists.w3.org/Archives/Public/public-web-perf/
 </pre>
 
 <pre class="anchors">
-text: DOMHighResTimeStamp; type: typedef; urlPrefix: http://www.w3.org/TR/hr-time/
-text: margin; type: attribute; url: http://www.w3.org/TR/CSS21/box.html#propdef-margin
-text: length; type: attribute; url: http://www.w3.org/TR/css3-values/#lengths
+urlPrefix: http://www.w3.org/TR/hr-time/; type: typedef; text: DOMHighResTimeStamp
+url: http://www.w3.org/TR/CSS21/box.html#propdef-margin; type: attribute; text: margin
+url: http://www.w3.org/TR/css3-values/#lengths; type: attribute; text: length
```

Comment:
Similarly, "length" is not an IDL attribute. It look like you want the `<length>` production - autolink that with `<<length>>`.